core/preview.ts

```diff
@@ -117,8 +116,14 @@ function addCandidate(url: string, candidate: PreviewCandidate): void {
 
   $links.setKey(url, { state: 'processed' })
   $candidates.set([...$candidates.get(), candidate])
-  if ($candidates.get().length === 1) {
-    if (!isMobile.get()) setPreviewCandidate(url)
+  if ($candidates.get().length === 1 && !isMobile.get()) {
+    getEnvironment().openRoute({
+      params: {
+        candidate: candidate.title,
```

Comment:
Why not URL?

Reply:
For aesthetic reasons, so that the URL looks good in the browser, for example: `/add/https%3A%2F%2Fdev.to%2Ffeed%2F/DEV%2520Community`. But now I think it has potential problem if two candidates will have same titles. So should I use here `candidate.url`?

Reply:
Yes, I think URL is safer. Title can be empty or non uniwue.